slices/python3.12-venv.yaml

```diff
@@ -0,0 +1,10 @@
+package: python3.12-venv
+
+slices:
+  libs:
+    essential:
+      - python3.12_standard
+      - python3-pip-whl_libs
+      - python3-setuptools-whl_libs
+    contents:
+      /usr/lib/python3.12/ensurepip/*.py:
```

Comment:
Please include the copyright slice below as well. Just list the symlink as it comes. That should do it.

Reply:
there was no copyright file in this package. I use dpkg-query -L to see file list of installed files.

Reply:
please see https://packages.ubuntu.com/noble/amd64/python3.12-venv/filelist

Reply:
Well, it has `./usr/share/doc/python3.12-venv -> python3.12` which points to python3.12's copyright. So you can do something like 

```yaml
/usr/share/doc/python3.12-venv: {symlink: /usr/share/doc/python3.12/copyright}
# or even
# /usr/share/doc/python3.12-venv:
# but that could bring additional file that we don't need
```

Reply:
P.S. you need to have it on the dedicated `copyright` slice